code/game/turfs/turf.dm

```diff
@@ -45,7 +45,12 @@
 		queue_smooth(src)
 
 /turf/Initialize(mapload)
-	. = ..()
+	if(initialized)
+		stack_trace("Warning: [src]([type]) initialized multiple times!")
```

Comment:
`stack_trace()` does not halt execution, dont know if this is intended or not

Reply:
Intended; we want want this logged; not halted.

atom initializations are the same.